docs/next.config.ts

```diff
@@ -263,6 +262,21 @@ export default withDocsInfra({
         destination: 'https://base-ui.com',
         permanent: true,
       },
+      {
```

Comment:
imo we can rather work towards removing the Next.js redirects. they have no effect on our prod setup. It's enough to rely on netlify _redirects

Reply:
Fair enough, I saw it was added for the /base_ui/ url so I thought why not, but I can remove it 👌